src/Components/Address/AddressForm.tsx

```diff
@@ -238,10 +238,10 @@ export const AddressForm: React.FC<AddressFormProps> = ({
       </Column>
       <Column span={12}>
         {/* Render the autocomplete optimistically to avoid an SSR mismatch */}
-        {(!billing && !autocomplete.loaded) || autocomplete.enabled ? (
+        {!billing && (!autocomplete.loaded || autocomplete.enabled) ? (
           <AutocompleteInput<AddressAutocompleteSuggestion>
             tabIndex={tabIndex}
-            disabled={!autocomplete.loaded}
+            loading={!autocomplete.loaded}
```

Comment:
This is the whole change. I'm going to go ahead and merge since I couldn't find a way to test it.